clustering/common/src/main/java/org/jboss/as/clustering/controller/PropertiesAttributeDefinition.java

```diff
@@ -70,7 +89,7 @@ public void marshallAsElement(AttributeDefinition attribute, ModelNode model, bo
                     writer.writeAttribute(Element.NAME.getLocalName(), property.getName());
                     // TODO if WFCORE-4625 goes in, use the util method.
                     String content = property.getValue().asString();
-                    if (content.indexOf('\n') > -1) {
+                    if (content.indexOf(System.lineSeparator()) > -1) {
```

Comment:
Is this change intentional? Because this would be checking for "\r\n" which wouldn't be the case when a unix file was parsed.

Reply:
Strange - I do not recall changing this.
Alas, we should delegate to AttributeMarshalller.marshallElementContent(...) instead.

Reply:
Updated.